exporters/otlp/protocoldriver.go

```diff
@@ -51,6 +51,25 @@ type ProtocolDriver interface {
 	ExportTraces(ctx context.Context, ss []*tracesdk.SpanSnapshot) error
 }
 
+// Provides options for setting up a split driver
+type Option func(d *splitDriver)
+
+// Metric driver allows one to set the driver used for metrics
+// in a SplitDriver
+func MetricDriver(dr ProtocolDriver) Option {
```

Comment:
```suggestion
// WithMetric driver allows one to set the driver used for metrics
// in a SplitDriver
func WithMetricDriver(dr ProtocolDriver) Option {
```

See the [Option naming guidelines](https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#options)